test/pipelinerun_test.go

```diff
@@ -322,10 +345,12 @@ func getPipelineRunSecret(suffix int, namespace string) *corev1.Secret {
 }
 
 func getHelloWorldPipelineRun(suffix int, namespace string) *v1alpha1.PipelineRun {
-	return tb.PipelineRun(getName(pipelineRunName, suffix), namespace, tb.PipelineRunSpec(
-		getName(pipelineName, suffix),
-		tb.PipelineRunServiceAccount(fmt.Sprintf("%s%d", saName, suffix)),
-	))
+	return tb.PipelineRun(getName(pipelineRunName, suffix), namespace,
+		tb.PipelineRunLabel("hello-world-key", "hello-world-value"),
```

Comment:
This just adds a custom label to the `service account propagation` test. @shashwathi is that what you had in mind, or do you think I should create a separate test for labels specifically? (I wasn't sure how conservative we wanted to be about adding new e2e tests)

Reply:
This looks good. Thank you @dwnusbaum